tests/network/probes.go

```diff
@@ -193,7 +195,7 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 				vmi = createReadyAlpineVMIWithLivenessProbe(livenessProbe)
 			} else if !isExecProbe(livenessProbe) {
 				By(specifyingVMLivenessProbe)
-				vmi = createReadyAlpineVMIWithLivenessProbe(livenessProbe)
+				vmi = createReadyAlpineVMIWithLivenessProbe(undoPointIpv6ProbeToSupportPod(livenessProbe))
```

Comment:
Despite making the tests hard to follow and somewhat unpredictable, does the fact Prob{} instances are being reused in the test introduces some bug?